internal/middleware/dumper.go

```diff
@@ -15,8 +15,8 @@ import (
 	"strings"
 )
 
-// responseDupper tees the response to a buffer and a response writer.
-type responseDupper struct {
+// responseDumper tees the response to a buffer and a response writer.
+type responseDumper struct {
```

Comment:
please revert this change. this struct is not responsible for the "dumping", it is responsible for "DUPlication" of the response. the dumper (after which the file is named) is the middleware. The difference in naming is intentional and shall not be changed.

also, it is out of scope for this MR.